### PR TITLE
[TRAFODION-3246] add slf4j in jdbc

### PR DESCRIPTION
**Pull request by mashengchen**
add a new jar which can use slf4j to print log

### CODE REVIEW

**Comment by DaveBirdsall** [comment]
@arvind-narain, @hegdean , @svarnau: Any comments on this one?

**Comment by svarnau** [comment]
This seems to be bundling in slf4j with JDBC driver.  If that is the case, then the slf4j license, which is already in licenses/lic-server-bin needs to also be added to licenses/lic-clients-bin.

**Comment by mashengchen** [comment]
thank you for your help Steve

**Comment by DaveBirdsall** [comment]
@svarnau, does this look good to you now?

File: licenses/lic-clients-bin [hunk 1]
```diff
@@ -32,3 +32,30 @@ MIT like license. (http://site.icu-project.org/)
   this Software without prior written authorization of the copyright holder.
 
 ===============================================================================
+The binary distribution of Apache Trafodion DCS and REST components bundle
```
Comment (svarnau) [inline licenses/lic-clients-bin]: here we should say JDBC driver instead of "DCS and REST components"

**Comment by DaveBirdsall** [comment]
@svarnau, does this look good to you now?

**Pull request merged**